web/vtadmin/src/components/inputs/Select.tsx

```diff
@@ -135,6 +137,7 @@ export const Select = <T,>({
     return (
         <div className={containerClass}>
             <Label {...getLabelProps()} label={label} />
+            <div className="mt-[-4px] mb-4">{description}</div>
```

Comment:
If description is optional, should we have a conditional checking for its existence here?

Reply:
Ahhh yeah, good catch